webpack.config.js

```diff
@@ -3,19 +3,21 @@ module.exports = {
   entry: "./webpack/entry.js",
   output: {
     // we're going to put the generated file in the assets folder so jekyll will grab it.
-      path: 'src/assets/javascripts/',
-      filename: "bundle.js"
+    path: __dirname + 'src/assets/javascripts/',
```

Comment:
Trying GitHub's new "suggestion" feature for simple, quick addition of this (from discussion https://github.com/allizad/jekyll-webpack/pull/11#issuecomment-430295873):
```suggestion
path: __dirname + '/src/assets/javascripts/',
```

Reply:
Nice, will fix. :)